### ISSUE TITLE
CockroachDB: error while creating table with comment

### ISSUE THREAD

**Issue by lucarestagno** [hasura/graphql-engine#9008]
How to reproduce.

* Add a cockroachDB data source
* Click on the schema
* Create a new table
* Fill the table name, table comment, and define at least one column
* Click on Create

We get an error from the backend:

<details>

Request

```json
{
  "type": "bulk",
  "source": "cdb",
  "resource_version": 8,
  "args": [
    {
      "type": "cockroach_run_sql",
      "args": {
        "source": "cdb",
        "sql": "CREATE TABLE \"public\".\"MyUsers\" (\"id\" serial NOT NULL, \"name\" Text NOT NULL, PRIMARY KEY (\"id\") );COMMENT ON TABLE \"public\".\"MyUsers\" IS E'Users tables';",
        "cascade": false,
        "read_only": false
      }
    }
  ]
}
```

Response

```json
{
  "code": "unexpected",
  "error": "database query error",
  "internal": {
    "arguments": [
      "(Oid 114,Just (\"[{\\\"oid\\\":109},{\\\"oid\\\":104},{\\\"oid\\\":111},{\\\"oid\\\":105},{\\\"oid\\\":107},{\\\"oid\\\":110}]\",Binary))",
      "(Oid 114,Just (\"[]\",Binary))"
    ],
    "error": {
      "description": null,
      "exec_status": "FatalError",
      "hint": null,
      "message": "unknown oid type: 114",
      "status_code": "08P01"
    },
    "prepared": true,
    "statement": "\n    SELECT\n      COALESCE(\n        ( SELECT\n            json_agg(\n              row_to_json(\n                (\n                  SELECT e\n                    FROM ( SELECT \"table\".relname AS \"name\",\n                                  \"schema\".nspname AS \"schema\"\n                    ) AS e\n                )\n              )\n            ) AS \"item\"\n            FROM jsonb_to_recordset($1::jsonb) AS oid_table(\"oid\" int)\n                 JOIN pg_catalog.pg_class \"table\" ON (\"table\".oid = \"oid_table\".oid)\n                 JOIN pg_catalog.pg_namespace \"schema\" ON (\"schema\".oid = \"table\".relnamespace)\n        ),\n        '[]'\n      ) AS \"tables\",\n\n      COALESCE(\n        ( SELECT\n            json_agg(\n              row_to_json(\n                (\n                  SELECT e\n                    FROM ( SELECT \"function\".proname AS \"name\",\n                                  \"schema\".nspname AS \"schema\"\n                    ) AS e\n                )\n              )\n            ) AS \"item\"\n            FROM jsonb_to_recordset($2::jsonb) AS oid_table(\"oid\" int)\n                 JOIN pg_catalog.pg_proc \"function\" ON (\"function\".oid = \"oid_table\".oid)\n                 JOIN pg_catalog.pg_namespace \"schema\" ON (\"schema\".oid = \"function\".pronamespace)\n        ),\n        '[]'\n      ) AS \"functions\"\n  "
  },
  "path": "$[0]"
}
```

Running he SQL statement on the CockroachDB instance:

```
root@localhost:26257/defaultdb> CREATE TABLE "public"."MyUsers" ("id" serial NOT NULL, "name" Text NOT NULL, PRIMARY KEY ("id") );COMMENT ON TABLE "public"."MyUsers" IS E'Users tables';
NOTICE: using sequential values in a primary key does not perform as well as using random UUIDs. See https://www.cockroachlabs.com/docs/v22.2/serial.html
CREATE TABLE
COMMENT ON TABLE

Note: timings for multiple statements on a single line are not supported. See https://go.crdb.dev/issue-v/48180/v22.2.
```

</details>

**Comment by danieljharvey** [comment]
This should be resolved as of #6046

**Comment by RG7279805** [comment]
@danieljharvey i am getting the same error on [v2.15.0].  Could you help me out
